rocksdb_admin/admin_handler.cpp

```diff
@@ -469,9 +469,23 @@ void AdminHandler::async_tm_addDB(
 
   // add the db to db_manager
   std::string err_msg;
+  replicator::DBRole role = replicator::DBRole::SLAVE;
+  if (request->__isset.db_role) {
+    if (request->db_role == "SLAVE") {
+      role = replicator::DBRole::SLAVE;
+    } else if (request->db_role == "MASTER") {
+      role = replicator::DBRole::MASTER;
+    } else if (request->db_role == "NOOP") {
+      role = replicator::DBRole::NOOP;
+    } else {
+      // Default behavior is to use SLAVE, so keep that
+      LOG(INFO) << "Invalid role requested to addDB " << request->db_role;
```

Comment:
Can we set the role default to be SLAVE, and only change it to be NOOP if db_role in the request is NOOP. Just wanted to minimize the changes to the code paths of other use cases.

Reply:
sounds good, just removed ability to set master